src/components/UpdateVotes.jsx

```diff
@@ -0,0 +1,46 @@
+import icon from "../assets/like.jpg";
+import iconDislike from "../assets/dislike.png";
+import * as Api from "../utils/newsApi";
+import { useState } from "react";
+
+export const UpdateVotes = ({ singleArticle, article_id }) => {
+  const [upVote, setUpVote] = useState(0);
+
+  const handleLikeClickVote = () => {
+    setUpVote((currVote) => currVote + 1);
+    Api.patchVotes(article_id, 1).catch((err) => {
+      if(err){
+        setUpVote((currVote) => currVote - 1);
+        return <p>Failed...</p>
+      }
+     
+    });
+  };
+
+  const handleDislikeClickVote = () => {
+    setUpVote((currVote) => currVote - 1);
+    Api.patchVotes(article_id, 1).catch((err) => {
```

Comment:
Here you should patch with -1, as currently you are still upvoting instead of downvoting.